components/CreatCatProfile/index.js

```diff
@@ -79,13 +79,18 @@ export default function CreateCatForm() {
           onChange={handleChange}
         />
         <label>Gender:</label>
-        <input
+        <select
```

Comment:
you should connect your label and your select using an `htmlFor` attribute for the label and an `id` for the select